src/aux-server/aux-web/shared/vue-components/UploadServerModal/UploadServerModal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <md-dialog :md-active.sync="isOpen" class="bot-upload-dialog">
         <md-dialog-title>Upload Files</md-dialog-title>
-        <div class="bot-upload-container">
+        <div class="bot-upload-container" style="max-height: 400px; overflow-y: auto">
```

Comment:
@TroyceGowdy Can you place these styles in UploadServerModal.css and use the `.bot-upload-container` CSS class?